Example/__tests__/Animation.test.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { renderHook } from '@testing-library/react-hooks';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  JestReanimated,
+} from 'react-native-reanimated';
+const {
+  withReanimatedTimer,
+  moveAnimationByTime,
+  moveAnimationByFrame,
+  getAnimatedStyle,
+} = JestReanimated;
+
+export default function TestComponent1(props) {
+  const widthSV = props.sharedValue ? props.sharedValue : useSharedValue(0);
```

Comment:
You mean this `if` condition or generally define shared value outside component?
If you mean the first option, I did this because I didn't want to copy and paste all component code with one different line. If you mean the second option, I did this because this was required in an example use case from docs.

Reply:
I mean using hooks in conditions, React strongly discourages from doing that.

Reply:
ok, I'll change this.